app/src/main/java/mozilla/lockbox/view/WelcomeFragment.kt

```diff
@@ -28,12 +32,35 @@ class WelcomeFragment : Fragment(), WelcomeView {
         val appLabel = getString(R.string.app_label)
         view.textViewInstructions.text = getString(R.string.welcome_instructions, appLabel)
         view.lockwiseIcon.contentDescription = getString(R.string.app_logo, appLabel)
+
         return view
     }
 
-    override val getStartedClicks: Observable<Unit>
+    override fun showExistingAccount(email: String) {
+        view?.apply {
```

Comment:
Is the view ever going to be null here?

Reply:
No, almost certainly not. I used this because it was the neatest way I could think off to not explicitly deref view repeatedly to get to the buttons. (i.e. `view!!.buttonGetStarted` and `view!!.buttonGetStartedManually`)